### PR TITLE
add `AsyncKernelClient` to `doc/api/client.rst`

### PR DESCRIPTION
**Pull request by helioz11**

### CODE REVIEW

**Review by davidbrochart — COMMENTED** [review]
Thanks @helioz11.
I'm wondering if it's worth adding the API of `AsyncKernelClient`, since it's the same as `BlockingKernelClient` except that it's async.
Maybe we should have the complete API for `BlockingKernelClient` and just say that `AsyncKernelClient` is identical but async?

**Comment by helioz11** [comment]
Sure, how about if I move the `automethod` from `AsyncKernelClient` to `BlockingKernelClient` and add extra information in `AsyncKernelClient`?

**Comment by davidbrochart** [comment]
Yes, that sounds good.

File: docs/api/client.rst [hunk 1]
```diff
@@ -49,3 +47,7 @@ client - communicating with kernels
    .. automethod:: wait_for_ready
 
    .. automethod:: is_alive
+
+.. autoclass:: AsyncKernelClient
+
+   The available class methods are identical with :class:`BlockingKernelClient` but async.
```
Comment (davidbrochart) [inline docs/api/client.rst:53]:
```suggestion
   :class:`AsyncKernelClient` is identical to :class:`BlockingKernelClient` but the methods described above are async.
```

**Review by davidbrochart — CHANGES_REQUESTED** [review]

**Review by davidbrochart — APPROVED** [review]
Thanks @helioz11.

**Pull request merged**